ha/setup/ha_setup.py

```diff
@@ -230,12 +247,33 @@ def __init__(self, args):
         Init method.
         """
         super().__init__(args)
+        self._ha_conf_index = "ha_update_index"
 
     def process(self):
         """
         Process init command.
         """
-        pass # TBD
+        Log.info("Processing init command")
+        Log.info("INIT: Update ha configuration files")
+        Conf.load(self._ha_conf_index, f"yaml://{const.HA_CONFIG_FILE}")
+        minion_name = self.get_minion_name()
+        node_type = Conf.get(self._index, f"cluster.{minion_name}.node_type").strip()
+        # Set env for cluster manager
+        self._update_env(node_type)
+        Log.info("INIT: HA configuration updated successfully.")
+        Log.info("init command is successful")
+
+    def _update_env(self, node_type):
+        """
+        Update env like VM, HW
+        """
+        Log.info(f"Detected {node_type} env.")
+        if "VM" == node_type.upper():
+            Conf.set(self._ha_conf_index, "CLUSTER_MANAGER.env", node_type.upper())
+        else:
+            # TODO: check if any env available other than vm, hw
```

Comment:
TODO missed or left intentionally?

Reply:
it is intentionally. Currently i checked all possible value and only provided is VM any there can be multiple env available for future so added TODO